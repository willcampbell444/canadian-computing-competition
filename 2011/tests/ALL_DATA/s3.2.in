6
4 240 108
4 262 212
4 299 274
4 299 275
4 81 221
3 59 114
