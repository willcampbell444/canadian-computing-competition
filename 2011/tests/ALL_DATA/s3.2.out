crystal
crystal
crystal
empty
empty
empty
